packages/announcements/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Please update this name to "Shiksha Announcements"

Reply:
Also use logo files logo192.png and logo512.png frm teachersapp package.

Reply:
Done